pom.xml

```diff
@@ -1339,5 +1339,27 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <name>hudson.Main.development</name>
+                <value>true</value>
```

Comment:
> we will have `hudson.Main.development=true` during JTH tests. I don't see any problem in that, is there?

Mm, not sure. JTH will set `Main.isUnitTest`. `getDefaultInstallState` currently checks that before `hudson.Main.development`, so we would not be changing behavior inside JTH, but you would need to check carefully that this has been true since 2.0, and preferably leave a comment in core noting the relative priority of these flags. OTOH if you can be sure that this configuration will only apply to `hpi:run`, and not `surefire:test`, then it should be harmless to set the Maven property.